feeadjuster/feeadjuster.py

```diff
@@ -204,10 +267,23 @@ def init(options: dict, configuration: dict, plugin: Plugin, **kwargs):
 plugin.add_option(
     "feeadjuster-threshold",
     "0.05",
-    "Channel balance update threshold at which to trigger an update. "
+    "Relative channel balance update threshold at which to trigger an update. "
     "Note: it's also fuzzed by 1.5%",
     "string"
 )
+plugin.add_option(
+    "feeadjuster-threshold-abs",
+    "100000000",
+    "Absolute channel balance update threshold (in msat) at which to trigger an update. "
+    "Note: it's also fuzzed by 1.5%",
+    "string"
+)
+plugin.add_option(
+    "feeadjuster-enough-liquidity",
+    "2000000000",
+    "Beyond this liquidity threshold (in msat) there is no need to adjusting fees",
+    "string"
+)
```

Comment:
1. Can this also be an msat string. i.e.  `0.02btc` ? This would be better for most users.
2. 2000000000 is ~400USD which is already quite big. But its hard to get a meaningful default on magic numbers that are prone to external factors (network state, fiat price, ...)